cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
The first thing you could check is if #1703 works on Firefly motes, e.g., by running examples/llsec/ipv6-demos/simple-udp-rpl. Since I only have OpenMotes, I can not try this on my own. Furthermore, if you enabled the debug output in potr.c and secmac.c, I might be able to identify the problem.